core/kroma_migration.go

```diff
@@ -205,3 +205,30 @@ func DeleteStateChanges(db ethdb.KeyValueStore, blockNumber uint64) error {
 	}
 	return batch.Write()
 }
+
+func DeleteAllStateChanges(db ethdb.KeyValueStore) error {
```

Comment:
If FinalizeTransition does bulk deletion, can we remove the logic to DeleteStateChanges every time in applyNewStateTransition?

Reply:
Rather I thought that we may not have to bulk delete, since `applyNewStateTransition` can cover all newly created stateChanges in the db. Hdyt @Pangssu?

Reply:
Stored state changes were designed to be deleted immediately after migrate.
Since code was added to wait before starting first migration, bulk deletion of state changes became necessary.
During the migration waiting period, unsafe blocks continue to increase. If geth is restarted during at that time, the stored state changes will not be used by the migrator.